autopts/ptsprojects/stack/layers/bap.py

```diff
@@ -129,15 +129,29 @@ def wait_scan_delegator_found_ev(self, addr_type, addr, timeout, remove=False):
 
     def wait_broadcast_receive_state_ev(self, broadcast_id, peer_addr_type, peer_addr,
                                         broadcaster_addr_type, broadcaster_addr,
-                                        pa_sync_state, timeout, remove=False):
+                                        pa_sync_state, advertiser_sid=None,
+                                        big_encryption=None, subgroups=None,
+                                        timeout=10, remove=False):
+        def _matches(expected, value):
+            if expected is None:
+                return True
+            if isinstance(expected, WildCard):
+                return True
+            return expected == value
```

Comment:
```suggestion
            return expected == value
```

Looking at the implementation of `WildCard`, I think we can skip the extra check here, as comparing with a wildcard object with `==` always returns `True`

Reply:
OK to keep as is